__tests__/src/components/GalleryView.test.js

```diff
@@ -19,28 +18,27 @@ function createWrapper(props) {
 
 describe('GalleryView', () => {
   let setCanvas;
-  let wrapper;
   beforeEach(() => {
     setCanvas = jest.fn();
-    wrapper = createWrapper({ setCanvas });
   });
   it('renders the component', () => {
-    expect(wrapper.find(Paper).length).toBe(1);
-    expect(wrapper.find(Paper).prop('component')).toEqual('section');
+    createWrapper({ setCanvas });
+    const buttons = screen.queryAllByRole('button');
+    expect(buttons[0].closest('section')).toBeInTheDocument(); // eslint-disable-line testing-library/no-node-access
```

Comment:
Assuming we want to check for a `<section>`, maybe we can just do:

```suggestion
    const { container } = createWrapper({ setCanvas });
    expect(container.querySelector('section')).toBeInTheDocument(); // eslint-disable-line testing-library/no-node-access, testing-library/no-container
```